account_service/manifest.xml

```diff
@@ -1,6 +1,6 @@
 <package>
   <name>account_service</name>
-  <version>1.0.1.0</version>
+  <version>1.0.2.0</version>
```

Comment:
You adjusted the `items_service` but increased the version of the `account_service`. Reset this and bump the `items_service` instead.